encode.go

```diff
@@ -101,8 +101,15 @@ func (e *Encoder) Encode(v interface{}) error {
 		e.buffer = encInt64(e.buffer, int64(val))
 
 	case time.Time:
+		if ZeroDate == val {
+			e.buffer = encNull(e.buffer)
+		} else {
+			e.buffer = encDateInMs(e.buffer, &val)
+			// e.buffer = encDateInMimute(v.(time.Time), e.buffer)
+		}
+
+	case *time.Time:
```

Comment:
`*time.Time` not cover all situation, should move to `default` case and switch after `UnpackPtrType`

Reply:
v.(type)  is *time.Time and kind is struct ,if move default, i still deal with *time.Time

Reply:
What is the uncovered situation you said？

Reply:
already push my code, pls help me to review this code.